### PR TITLE
Calendar was created

### PR DESCRIPTION
**Pull request by dariia-romanova**
- [DEMO LINK](https://dariia-romanova.github.io/layout_calendar/)

### CODE REVIEW

**Comment by darynadudar** [comment]
Why are there only 6 days of the week?

image

**Review by darynadudar — CHANGES_REQUESTED** [review]
Everything is perfect, just fix days of week

**Review by darynadudar — APPROVED** [review]
🔥🔥🔥

File: src/styles/blocks/calendar.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $day-width;
+    height: $day-width;
+    border: 1px solid #ccc;
+    background-color: #eee;
+    transition: all 0.5s ease-in-out;
```
Comment (aionova) [inline src/styles/blocks/calendar.scss]: indicate which property you use transition for. Don't use transition all

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        content: "#{$i}";
+        font-family: Arial, sans-serif;
+        font-size: 30px;
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: #f3dde3;
+      // position: relative;
```
Comment (aionova) [inline src/styles/blocks/calendar.scss]: remove comments

**Review by aionova — APPROVED** [review]

**Pull request closed without merging**